server/controllers/createUser.js

```diff
@@ -1,62 +1,137 @@
 const User = require('../database/models/User');
 
-module.exports = app => {
-  // Sign Up
-
-  app.post('/db/account/signup', (req, res, next) => {
```

Comment:
for what the``` next``` it's not a middleware!

Reply:
havent dealt with it yet, but will use it tomorrow

Reply:
will be using the next in another step. there should be a comment in the code.